joomla/joomla-cms
# [4.0] [a11y] Make ActionButton Featured accessible

### Summary of Changes
Action Buttons for featuring articles for example in com_content/articles are Links, not Buttons.
This PR
- changes the a tag to button tag
- removes popver and replaces it by a sr-only class
- makes a separate column in the articles list 
- enables sorting the list by featured articles - see also #23497

### Testing Instructions
Apply the patch and see that the featured, unfeatured function works as before. 

![featured-column-articles](https://user-images.githubusercontent.com/1035262/51972689-4ee9ca80-247c-11e9-944b-37df43fab815.PNG)

NOTE:
When activating one featured button, the corresponding checkbox and the Status Change Dropdown are activated for a moment. #23717
### Documentation Changes Required
new screen
